prody/database/pfam.py

```diff
@@ -473,36 +475,46 @@ def fetchPfamPDBs(**kwargs):
                 raise ValueError('Please provide an integer for start or end when using query.')
 
     from ftplib import FTP
-    data = []
+    data_file = open('pdbmap.gz','wb')
```

Comment:
This will generate a file on user’s working directory. If this file is intermediate, it is better to avoid to generate it. Please consider using file stream if there is no workaround.

Reply:
I don't know how to use a stream from a gz file.

Reply:
It actually looks like there is no option to do that.